docs/buildah-build.1.md

```diff
@@ -411,6 +411,24 @@ By default, Buildah manages _/etc/hosts_, adding the container's own IP address.
 
 Set the OS of the image to be built, and that of the base image to be pulled, if the build uses one, instead of using the current operating system of the host.
 
+**--output**, **-o**=""
+
+Output destination (format: type=local,dest=path)
+
+The --output (or -o) option extends the default behavior of building a container image by allowing users to export the contents of the image as files on the local filesystem, which can be useful for generating local binaries, code generation, etc.
+
+The value for --output is a comma-separated sequence of key=value pairs, defining the output type and options.
+
+Supported _keys_ are:
+- **dest**: Destination path for exported output. Valid value is absolute or relative path.
+- **type**: Defines the type of output to be used. Valid values is documented below.
+
+Valid _type_ values are:
+- **local**: write the resulting build files to a directory on the client-side.
+- **tar**: write the resulting files as a single tarball (.tar).
+
+If no type is specified, the value defaults to local. Use a hyphen as non-comma format (`-`) to write the output tarball to standard output (`STDOUT`).
```

Comment:
This
- doesn’t document `dest=-` AFAICS, although the implementation now supports that.
- Says “non-comma format”, which is a short-hand I started using for the purposes of this review, but that is unclear without extra content.


Maybe something vaguely like
> ```
> - **dest**: … absolute or relative path; or, as a special case, `-` means the standard output.
> …
> Alternatively, instead of a comma-separated sequence, the value of **--output** can be just a destination (in the `**dest** format) (e.g. `--output some-path`, `--output -`)
> ```

Reply:
The “alternatively” part should also document the implied `type` value…

… wait, `--output foo` means `type=local`, but `--output -` means `type=tar`? That needs to be explicitly documented as well.